lib/videoroom_web/stats_channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule VideoRoomWeb.StatsChannel do
+  @moduledoc """
+  A secondary channel used to send metrics reports to the frontend
+  """
+  use Phoenix.Channel
+
+  @impl true
+  def join("stats", _message, socket) do
+    Phoenix.PubSub.subscribe(VideoRoom.PubSub, "metrics")
+    {:ok, socket}
+  end
+
+  @impl true
+  def handle_info({:metrics, stats}, socket) do
+    push(socket, "metrics", %{stats: stats})
+    {:noreply, socket}
+  end
```

Comment:
There's no need to subscribe to PubSub manually, this channel already subscribes to `stats`
```suggestion
  def join("stats", _message, socket) do
    {:ok, socket}
  end
```

Reply:
Ah, this explains why I was getting duplicated messages when I subscribed to "stats" instead of "metrics" :+1: